lib/active_model/serializer/adapter.rb

```diff
@@ -80,11 +81,11 @@ def self.inherited(subclass)
         ActiveModel::Serializer::Adapter.register(subclass.to_s.demodulize, subclass)
       end
 
-      attr_reader :serializer
+      attr_reader :serializer, :instance_options
 
       def initialize(serializer, options = {})
         @serializer = serializer
-        @options = options
+        @instance_options = options
```

Comment:
should this be `self.instance_options` like the other `self` accesses? 
or is the `self` usage only for `_`'ed attributes?